### ISSUE TITLE
a bug in gcollapse

### ISSUE THREAD

**Issue by ractingmatrix** [mcaceresb/stata-gtools#27]
hi, it seems there is a bug - please refer to the following:
*************************************************
clear *
set obs 100
gen aaaxp=1
gen aaa=1
gcollapse (mean) aaaxp=aaaxp (max) aaax=aaa //error: variable aaaxp not found

clear *
set obs 100
gen aaaxp=1
gen aaa=1
collapse (mean) aaaxp=aaaxp (max) aaax=aaa //no error

**Comment by sergiocorreia** [comment]
Can also be replicated here:

```stata
sysuse auto, clear
gcollapse (mean) pric = price
gcollapse (mean) pric = price
```

And might be caused by this line:

```
rename `var' __gtools`__gtools_i'
```

**Comment by mcaceresb** [comment]
Technically that line is culprit, yes, but the fix is really simple.

```

. clear

. set obs 10
obs was 0, now 10

. gen xx = 1

. set varabbrev on

. gcollapse x = xx
variable xx not found
r(111);

. set varabbrev off

. gcollapse x = xx
```

The fix is to include `set varabbrev off` within gcollapse. I will modify the code tonight or tomorrow. In the meantime please run your code with `set varabbrev off` as a temporary fix.

**Comment by mcaceresb** [comment]
This should be fixed in `0.10.1` in the develop branch. I'll be merging to master soon.
